src/app.ts

```diff
@@ -319,16 +339,16 @@ function appInit(gopt: AppOpt = {}): App {
 		return app.mouseDeltaPos.clone();
 	}
 
-	function isMouseClicked(): boolean {
-		return app.mouseState === "pressed";
+	function isMousePressed(m = "left"): boolean {
```

Comment:
If you don't pass an argument to isKeyPressed(), it'll return true if any key was just pressed. Wonder if why should do the same here to return true if any mouse button was just pressed, or default to left?

Reply:
sounds like a bit of an edge case but why not just make this param required? seems confusing otherwise

Reply:
The assumption is people mainly deal with the left mouse button, maybe this can save some typing. Or maybe people will assume this function would check for left mouse by default?